src/app/story/artifact-relationship/artifact-relationship.component.spec.ts

```diff
@@ -12,7 +12,8 @@ import { relationships } from '../test.data';
 
 
 describe('ArtifactRelationshipComponent', () => {
-  let fixture: ComponentFixture<ArtifactRelationshipComponent>;
+
+  // let fixture: ComponentFixture<ArtifactRelationshipComponent>;
```

Comment:
Could you remove this line?

Reply:
Since this file now has nothing in it, should I rewrite the test instead, or maybe just delete the file? Originally it was testing build tags, which we removed

Reply:
@sarah256 could you modify the test to use a different artifact type? I want to keep unit test coverage on that component.